sc_preprocessing/2_make_pseudobulks/pseudobulk_synovium.ipynb

```diff
@@ -0,0 +1,1192 @@
+{
```

Comment:
<p>I keep it in aug_data_path, because in my mind, these are "results" needed to understand the augmented data. I could see arguments either way, but this is how I currently do it in my workflow.</p>